### DISCUSSION TITLE
Any workaround to update a file inline without changing the whitespace.

### DISCUSSION THREAD

**Discussion by Bennet-Sunder** [mikefarah/yq#913] [General]
Suppose I have a yml file like below, and I am doing some in place update to the file. The result joins the multiline file into a single one. With some huge file this might create a overhead to clean up even if it's the right thing. Is there a way we can avoid this problem and have only the updated value as the change in the original file. Or any workarounds to reduce this impact of whitespace. 
I did read about an existing issue with whitespace from the docs. But wanted to know if there is a way I can avoid it for now. 

```
---
root:
  key:
    first: name is this and
         it keeps going on
```

**Comment by mikefarah** [comment]
Not at the moment sorry :( it's done by the underlying yaml parser and I don't have any control of it.

**Reply by mikefarah** [reply]
That said - it only affects the readability of the yaml document - the actual content is the same

**Reply by Bennet-Sunder** [reply]
true the actual content is the same. However in huge yaml document the diff become too much to keep track of.

**Reply by hamza-louis** [reply]
Hello @mikefarah , is this possible now ? Can we use yq to change specific fields without re-writing the multi-line lists to a single line ?
